packages/graphql/tests/integration/enums.int.test.ts

```diff
@@ -87,4 +87,58 @@ describe("enums", () => {
             await session.close();
         }
     });
+
+    test("should create a movie (with a default enum)", async () => {
+        const session = driver.session();
+
+        const statusResolver = {
+            ACTIVE: "active",
+        };
```

Comment:
Interesting, I reckon we should add another test case **not** using the statusResolver (and `ACTIVE` being the expected result.

I would also consider adding this usage to the docs, as it is quite handy, and may not be obvious

Reply:
That makes sense!

Reply:
Ah, looking closer at these tests, I think they should cover all four cases: with/without default value, and with/without custom resolver. Added!

Reply:
mmm, I think that was not needed, as those cases were already covered in the tck (and integration covers the actual database translation) but having them won't hurt. Thanks!